docs/features/decorators.rst

```diff
@@ -69,33 +69,25 @@ This set of decorators defines test levels:
 
 * ``tier1`` marks component level functional tests (that verify defined functional requirements using a range of normal and erroneous input data). Example::
 
-    from robottelo.decorators import tier1
```

Comment:
Add :
```
Import pytest
```
wherever you have made the tier changes.

Reply:
Fixed